linux_pipeline/Jenkinsfile_ubuntu_azure_kernel_validation

```diff
@@ -27,20 +27,18 @@ def HYPERV_VHD_HASH = [trusty:'Ubuntu_14.04.5\\Ubuntu14.04.5-server.vhdx',
                   xenial:'ubuntu_16.04.5\\ubuntu_16.04.5.vhdx',
                   bionic:'ubuntu_18.04.1\\ubuntu_18.04.1_gen1_gen2.vhdx',
                   cosmic:'ubuntu_18.10\\ubuntu_18.10_gen1_gen2.vhdx']
-def HYPERV_VALIDATION_TESTS_HASH = [HV_BVT:"-TestCategory 'BVT'",
```

Comment:
are the HV_BVT, Azure_BVT used anywhere?

Reply:
no, below script using HYPERV_VALIDATION_TESTS_HASH.each, AZURE_VALIDATION_TESTS_HASH.each